cns/logger/log.go

```diff
@@ -104,6 +104,12 @@ func Debugf(format string, args ...interface{}) {
 	sendTraceInternal(msg)
 }
 
+func LogNCSnapshot(aiEvent aitelemetry.AiEvent) {
+	aiEvent.Properties[OrchestratorTypeStr] = Log.Orchestrator
```

Comment:
Any specific reason these 2 lines are not assigned inside TrackEvent function too?

Reply:
this is swift specific..should not be in aitelemetry package